scripts/log-serial/log-serial.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+
+import serial
+import sys
+from datetime import datetime
+
+
+if len(sys.argv) < 2:
+    print("Usage: ./log-serial.py <file_path>")
+    sys.exit(1)
+
+file_path = sys.argv[1]
+
+serport = serial.Serial(baudrate=19200)
+serport.setPort("/dev/serial/by-id/usb-FTDI_Quad_RS232-HS-if03-port0")
+serport.open()
+
+# Don't reset the MMC but drive the P2.10 pin to HIGH, so it can be reset
+# externally and execute the application code, not the ROM bootloader
+serport.setRTS(0)  # set RTS line to 3.3V
+serport.setDTR(0)  # set DTR line to 3.3V
+
+
+with open(file_path, "a") as log_file:
+    while True:
+
+        # There is more than 0 bytes in input buffer
+        if serport.in_waiting > 0:
+
+            message = serport.readline().decode("utf-8", errors="replace").strip()
```

Comment:
Exceptions must be handled here, if the serial port is disconnected, the application should finish gracefully.